interface/billing/sl_eob_search.php

```diff
@@ -615,7 +615,7 @@ function npopup(pid) {
 
    <table border='0' cellpadding='5' cellspacing='0'>
 
-     <tr bgcolor='#ddddff'>
+     <tr class = "tabs_body_light" bgcolor='#ddddff'>
```

Comment:
not sure applying special classes to code for a subset of themes is feasible. The goal is to keep classes as abstract as possible. This solution isn't maintainable long term over the whole codebase.

I'm not sure I see why this is needed. Only 1 (ok, technically 2) style sheets get brought in, the selected theme and core (core being theme agnostic only defining globally needed stuff). What is preventing you from just defining what you need in the CSS file? You wouldn't be affecting other themes with the flat style as those themes CSS files aren't rendered in the flat styles